src/house_package_saachi/main.py

```diff
@@ -0,0 +1,59 @@
+from src.house_package_saachi.data_ingestion import (
+    fetch_housing_data,
+    load_housing_data
+)
+from src.house_package_saachi.data_preprocessing import preprocess_housing_data
+from src.house_package_saachi.model_training import (
+    train_linear_regression,
+    train_decision_tree,
+    train_random_forest,
+    evaluate_model,
+)
+from src.house_package_saachi.model_scoring import score_model
+from src.house_package_saachi.utils import stratified_split
+import pandas as pd
+import numpy as np
+
+

```

Comment:
- This `main.py` must be kept outside `src/`.
- And all the _imports_ must be like `from house_package_saachi.data_ingestion import load_housing_data` and not `from src.house_package_saachi.data_ingestion import load_housing_data`.

Reply:
- Created main.py outside src/
- Made changes in main.py file accordingly